Sources/SectionScrubber.swift

```diff
@@ -18,6 +20,14 @@ public class SectionScrubber: UIView {
 
     static let RightEdgeInset: CGFloat = 5.0
 
+    /*
+     When calculating the NSIndexPath for the current scrubber position we need to use a x and a y coordinate,
+     the y coordinate is provided by how far you have scrubber the scrubber, meanwhile we use a hardcoded x since
+     using 5 would ensure us that most of the time the first item in each row will be selected to retreive the
+     index path at certain location.
+    */
+    static let initialXCoordinateToCalculateIndexPath = CGFloat(5)
```

Comment:
@marijnschilling middle of the screen might make it think there are no cells in a row like this. Any way I can make the comment more clear to describe that? That's what I was trying to explain.

![group](https://cloud.githubusercontent.com/assets/1088217/16734343/7d627014-4785-11e6-9fa4-633ed0055f49.png)